charts/hush-sensor/values.yaml

```diff
@@ -277,6 +277,13 @@ sentry:
     # Custom annotations for sentry service account
     annotations: {}
 
+  crawler:
+    aws:
+      # irsa, assume_role or empty string
```

Comment:
Need to document what each of the modes does.
If `assume_role` is not supported yet in the service remove it from the PR completely.